lib/css/components/_stacks-inputs.less

```diff
@@ -69,6 +70,9 @@
     //  --  REMOVE EDGE ADDITIONS
     //  ------------------------------------------------------------------------
     &::-ms-clear { display: none; }
+
+    // --  STYLE SCROLLBARS
+    @scrollbar-styles();
```

Comment:
This is potentially a breaking change too. Should see how this looks on scrollable inputs before merge.

Reply:
Checked :v: